lib/railgun/message.rb

```diff
@@ -0,0 +1,115 @@
+require 'mail'
+require 'mailgun/messages/message_builder'
+require 'railgun/attachment'
+require 'railgun/errors'
+
+module Mail
+
+  class Message
```

Comment:
Yes I am. It would have been better to extend Mail::Message but I'm not sure that was an option because of how ActiveMailer and all that work.